### PR TITLE
Determine p ratio

### LINKED ISSUE #19: research how the calories consumed and calorie surplus affect p-ratio
ditto

### PR DESCRIPTION
**Pull request by mentalcaries**
For this PR I:
- set up the data structure for p-ratios for sex/surplus/fitness level in `pRatios.ts`
- wrote functions to retrieve those values based on parameters for sex, fitnessLevel, bodyFat% and calorie surplus%
- set up a test route/page with form components to test the calculations at `/pratio`

Routes and server side code will still have to be properly setup.

Closes #19 

image

### CODE REVIEW

File: src/routes/pratio/+page.svelte [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+  $: if(sex === 'male'){
+    bodyFat = 10;
+    bodyFatMin = 4;
+    bodyFatMax = 20
+  }
+  else{
+    bodyFat = 22
+    bodyFatMin = 10;
+    bodyFatMax = 32;
```
Comment (thinklikeadesigner) [inline src/routes/pratio/+page.svelte]: bodyfat max for female is 40%

https://github.com/thinklikeadesigner/FlexFactor/issues/19
```suggestion
    bodyFatMax = 40;
```

File: src/routes/pratio/+page.svelte [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  let bodyFatMax = 20;
+
+  $: if(sex === 'male'){
+    bodyFat = 10;
+    bodyFatMin = 4;
+    bodyFatMax = 20
+  }
+  else{
+    bodyFat = 22
+    bodyFatMin = 10;
```
Comment (thinklikeadesigner) [inline src/routes/pratio/+page.svelte]: bodyfat min for female is 12%

https://github.com/thinklikeadesigner/FlexFactor/issues/19
```suggestion
    bodyFatMin = 12;
```

**Review by thinklikeadesigner — CHANGES_REQUESTED** [review]

File: src/routes/pratio/+page.svelte (hunk 1 above)
Comment (mentalcaries) [reply src/routes/pratio/+page.svelte]: Good catch !!

**Review by thinklikeadesigner — APPROVED** [review]
LGTM ✅

**Pull request merged**